diff --git a/17-3.5/Dockerfile b/17-3.5/Dockerfile
index 7af430b..6b5669f 100644
--- a/17-3.5/Dockerfile
+++ b/17-3.5/Dockerfile
@@ -2,7 +2,7 @@
 # NOTE: THIS DOCKERFILE IS GENERATED VIA "make update"! PLEASE DO NOT EDIT IT DIRECTLY.
 #
 
-FROM postgres:17-bullseye
+FROM docker.io/library/postgres:17-bullseye
 
 LABEL maintainer="PostGIS Project - https://postgis.net" \
       org.opencontainers.image.description="PostGIS 3.5.2+dfsg-1.pgdg110+1 spatial database extension with PostgreSQL 17 bullseye" \
